frontend/apps/reader/modules/readersearch.lua

```diff
@@ -227,6 +234,7 @@ function ReaderSearch:onShowFulltextSearchInput()
                 {
                     -- @translators Find all results in entire document, button displayed on the search bar, should be short.
                     text = C_("Search text", "All"),
+                    is_enter_default = true,
```

Comment:
```suggestion
```

Reply:
Is there an easy way to user patch this without having to copy the whole function? @poire-z

Reply:
I don't see another way than copying the whole function :/